plugins/gatsby-remark-embed-speakerdeck/index.js

```diff
@@ -0,0 +1,14 @@
+const visit = require('unist-util-visit')
+const sd = (_, id) => {
+  return `<script async class="speakerdeck-embed" data-id="${id}" data-ratio="1.77777777777778" src="https://speakerdeck.com/assets/embed.js"></script>`
```

Comment:
Not sure why, but this script tag doesn't download the JS when I run `gatsby develop`. I only get the embed working if I do a full build and then view the static site. Other HTML shows up, but no script tags execute.